test/asan-no-leak.js

```diff
@@ -1,5 +1 @@
-if (Module != undefined) {
-  Module.ASAN_OPTIONS = 'detect_leaks=0';
-} else {
-  Module = {ASAN_OPTIONS: 'detect_leaks=0'};
-}
+Module['ASAN_OPTIONS'] = 'detect_leaks=0:verbosity=1';
```

Comment:
Did this do anything? If this is an environment variable, I don't think we read those from `Module`?

Reply:
ASAN has specific code for reading ASAN_OPTIONS from the Module yes: https://github.com/emscripten-core/emscripten/blob/6416c351c6d98b796dfefd8a8d00d71e83ca7fa5/system/lib/compiler-rt/lib/asan/asan_flags.cpp#L128-L141

I didn't mean to a verbose=1 there though..

I

Reply:
I see, thanks!